app/(auth)/signin.js

```diff
@@ -27,12 +28,7 @@ export default function SignIn() {
 
   async function handleSignIn() {
     try {
-      const userCredential = await signInWithEmailAndPassword(
-        auth,
-        email,
-        password,
-      );
-      // console.log(userCredential.user);
+      await signInWithEmailAndPassword(auth, email, password);
```

Comment:
Is this just to get rid of the log? Or do we not want to store the user Credential?

Reply:
the userCredential local variable is not being used at all